src/__tests__/starWarsData.js

```diff
@@ -76,29 +79,39 @@ const artoo = {
 };
 
 const droidData = {
-  2000: threepio,
-  2001: artoo,
+  '2000': threepio,
+  '2001': artoo,
 };
 
+type CharacterJS = {
+  id: string,
+  name?: string,
+  friends?: string[],
+  appearsIn?: number[],
+};
 /**
  * Helper function to get a character by ID.
  */
-function getCharacter(id) {
+function getCharacter(id): Promise<*> {
   // Returning a promise just to illustrate GraphQL.js's support.
   return Promise.resolve(humanData[id] || droidData[id]);
 }
 
 /**
  * Allows us to query for a character's friends.
  */
-export function getFriends(character) {
+export function getFriends(character: CharacterJS): Array<*> {
+// as you can see, with Flow
+// If you choose a schema structure like {friends?: Array<*>}
+// You must check it , or re-design your schema
```

Comment:
nit: please align comments to the other lines in this body. also no space before comma